code/game/objects/items/weapons/stunbaton.dm

```diff
@@ -143,6 +143,8 @@
 		if(!bcell)
 			user.simple_message("<span class='warning'>[src] does not have a power source!</span>",
 				"<span class='warning'>[src] has no pulse and its soul has departed...</span>")
+		else if (bcell.maxcharge < hitcost)
+			user.simple_message("<span class='warning'>[src] clicks but nothing happens. You suspect something's wrong with the battery.</span>")
```

Comment:
I think "Something must be wrong with the battery" flows better than "You suspect something's wrong with the battery"

Reply:
use to_chat instead